packages/sirius-web/frontend/sirius-web/package.json

```diff
@@ -52,6 +53,7 @@
     "xstate": "4.32.1"
   },
   "devDependencies": {
+    "@rollup/plugin-image": "2.1.1",
```

Comment:
This shouldn't be necessary for this PR

Reply:
I will try again without, but maybe this change (or other dependency changes in this PR) should have been done in a previous PR.
What I mean is, sometimes, the team does not generate package-lock.json well, so the next dev who does it correctly, have to make the change that should have been done in a previous PR.